cfy_manager/components/validations.py

```diff
@@ -532,14 +537,18 @@ def _validate_external_postgres():
                         problems=' '.join(problems),
                     )
                 )
-        elif not pg_conf[SSL_ENABLED]:
-            raise ValidationError('When installing an external database, SSL'
-                                  ' must be enabled')
-
-    if _is_installed(MANAGER_SERVICE) \
-            and not config[POSTGRESQL_CLIENT][SSL_ENABLED]:
-        raise ValidationError('When using an external database, SSL'
-                              ' must be enabled')
+
+    if _is_installed(MANAGER_SERVICE) and not _is_installed(DATABASE_SERVICE):
+        if (
+            pg_client['host'] in ('127.0.0.1', 'localhost')
+            and pg_conf['cluster']['nodes']
```

Comment:
this is pretty much the same condition as before! utils function, then?

Reply:
Factored out into a utils function.